### PR TITLE
wip_started styling

### PR DESCRIPTION
**Pull request by atasker**

### CODE REVIEW

File: app/assets/stylesheets/full-width-pics.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+.image-bg-fixed-height {
+    text-align: center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    background-size: cover;
+    -o-background-size: cover;
+}
+
+.image-bg-fluid-height {
+	background: url('../assets/china.png') no-repeat center center scroll;
```
Comment (cpsoinos) [inline app/assets/stylesheets/full-width-pics.css:36]: over-indented

File: app/assets/stylesheets/full-width-pics.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    padding: 100px 0;
+    filter: blur(0px) grayscale(0%);
+}
+
+.image-bg-fixed-height {
+	background: url('../assets/china.png') no-repeat center center scroll;
```
Comment (cpsoinos) [inline app/assets/stylesheets/full-width-pics.css:42]: over-indented

File: app/assets/stylesheets/full-width-pics.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    height: 450px;
+    filter: blur(0px) grayscale(0%);
+}
+
+.img-center {
+	margin: 0 auto;
```
Comment (cpsoinos) [inline app/assets/stylesheets/full-width-pics.css:48]: over-indented

File: app/assets/stylesheets/full-width-pics.css [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+.rice-bowl {
+    width: 130px;
+    height: auto;
+    padding-bottom: 6px;
+}
+
+.right-border {
+    border-right: solid thin #999;
+}
+
```
Comment (cpsoinos) [inline app/assets/stylesheets/full-width-pics.css]: extra blank lines at the end of the file

File: app/models/restaurant.rb [hunk 5]
```diff
@@ -8,4 +8,20 @@ class Restaurant < ActiveRecord::Base
   validates :city, presence: true
   validates :state, presence: true
   validates :zip_code, presence: true
+
+  def average_rating
+    self.reviews.sum(:rating)/self.reviews.count
```
Comment (cpsoinos) [inline app/models/restaurant.rb:13]: `self` may be redundant here - check if there's the same functionality when removing self (I may be wrong here)

File: app/models/restaurant.rb [hunk 6]
```diff
@@ -8,4 +8,20 @@ class Restaurant < ActiveRecord::Base
...
+  end
+
+  def self.top_restaurants
+    top_hash = {}
+    top_array = []
+    restaurants = Restaurant.all
```
Comment (cpsoinos) [inline app/models/restaurant.rb:19]: add a limitation on Restaurant.all, maybe by page

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,31 +1,58 @@
...
+                  <%- end -%>
+                </ul>
+            </div>
+        <!-- /.navbar-collapse -->
+    </nav>
 
+    <!-- Full Width Image Header with Logo -->
+    <!-- Image backgrounds are set within the full-width-pics.css file. -->
+    <header class="image-bg-fluid-height">
+<!--         <img class="img-responsive img-center" src="http://placehold.it/200x200" alt="">
```
Comment (cpsoinos) [inline app/views/layouts/application.html.erb]: is this a placeholder line/do we need it here?

File: app/views/restaurants/index.html.erb [hunk 8]
```diff
@@ -1,31 +1,60 @@
...
-</table>
+  <%= paginate @restaurants %>
 
-<%= paginate @restaurants %>
+  <!-- Footer -->
+  <footer>
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+          <p>Copyright &copy; ACES 2015</p>
```
Comment (cpsoinos) [inline app/views/restaurants/index.html.erb:53]: haha nice!

File: app/models/restaurant.rb [hunk 9]
```diff
@@ -8,4 +8,20 @@ class Restaurant < ActiveRecord::Base
   validates :city, presence: true
   validates :state, presence: true
   validates :zip_code, presence: true
+
+  def average_rating
```
Comment (cpsoinos) [inline app/models/restaurant.rb:12]: some unit tests would be great here

File: app/models/restaurant.rb [hunk 10]
```diff
@@ -8,4 +8,20 @@ class Restaurant < ActiveRecord::Base
...
+    self.reviews.sum(:rating)/self.reviews.count
+  end
+
+  def self.top_restaurants
```
Comment (cpsoinos) [inline app/models/restaurant.rb:16]: some unit tests would be great here

File: app/views/layouts/application.html.erb (hunk 7 above)
Comment (atasker) [inline app/views/layouts/application.html.erb]: cleaned this up, but we still need the header.

File: app/assets/stylesheets/full-width-pics.css (hunk 1 above)
Comment (atasker) [inline app/assets/stylesheets/full-width-pics.css:36]: I'm pretty sure 2 tabs is standard for CSS, my editor doesn't seem to like anything less.

File: app/assets/stylesheets/full-width-pics.css (hunk 4 above)
Comment (atasker) [inline app/assets/stylesheets/full-width-pics.css]: got rid of these

File: app/models/restaurant.rb (hunk 5 above)
Comment (atasker) [inline app/models/restaurant.rb:13]: I played around with this, it doesn't perform properly without being declared on 'self', not 100% sure why though : )

Comment (cpsoinos) [inline app/models/restaurant.rb:13]: Cool - just wanted to check since Elliott and I ended up removing the calls to self on some of the voting methods

**Pull request merged**